README.md

```diff
@@ -1 +1,31 @@
-# angular-what-is-angular-readme
+# What is Angular and why does it exist?
+
+## Overview
+
+AngularJS (commonly referred to as just "Angular") is an open-source framework released in 2009, being mainly maintained by Google as well as a massive community of developers.
+
+It aims to solve the problems faced when creating single page applications, implementing a MVVM pattern to separate presentation and business logic.
+
+## Objectives
+
+- Describe what Angular's role in the web platform is
+- Describe the issues Angular looks to solve
+- Describe how Angular thinks in terms of Web Components to create encapsulation and reusable components
+
+## Traditional Rendering
```

Comment:
The first section here should probably be "called What's a Front End Framework?" What are some of the problems of not using a front end framework? The students will likely have seen some of them and be able to identify. For example, here's a paragraph from the introductory Rails README.

> Why Use a Framework?

> If you wanted to put surround sound in your house, would you go and spend years researching how to best fabricate speakers, learn how to transfer sound through wires, weld your own sound board, and create from scratch every other component that would be required to have a surround sound system in your home? Most likely not. Instead it would be much smarter and more efficient to purchase a surround sound system from an organization that already put in all of the research and development work to create a professional system.

> In the same way, when it comes to building a web application it would technically be possible to build out all of the functionalities yourself, but it's typically a better idea to leverage a system that has already spent over a decade developing the tools necessary for getting applications built.

Notice that it uses a real-world analogy that's simple and non-technical. This eases the student into the material in an accessible way. Don't be afraid to show your personality. 

This first section could look something like:

1.  What's a Front End Framework? 
    1. Describe the problems of not using a front end framework
        1. Pages can only be updated on refresh
        2. Javascript code becomes unwieldy and hard to manage
    2. Describe what a front end framework is and how it solves the problem

This section should be brief (5-6 sentences?) and as non-technical as possible (again, see the intro above.)